rios/riostests/teststats.py

```diff
@@ -57,7 +57,7 @@ def run():
         (gdal.GDT_Float64, 0.01)
     ]
     # Include 64-bit int types, if supported
-    if hasattr(gdal, 'GDT_UInt64'):
+    if VersionObj(gdal.__version__) >= VersionObj('3.5.2'):
```

Comment:
Maybe have a link here to that GDAL histogram PR for reference?

Reply:
Good thinking, thanks